e2e-tests/tfm/cypress/e2e/journeys/deal-cancellation/gef/submit-deal-cancellation-effective-from-in-past.spec.js

```diff
@@ -0,0 +1,121 @@
+import { TFM_DEAL_STAGE, TFM_FACILITY_STAGE } from '@ukef/dtfs2-common';
+import relative from '../../../relativeURL';
+import { ADMIN, BANK1_MAKER1, PIM_USER_1, T1_USER_1 } from '../../../../../../e2e-fixtures';
+import caseDealPage from '../../../pages/caseDealPage';
+import { today, yesterday } from '../../../../../../e2e-fixtures/dateConstants';
+import checkDetailsPage from '../../../pages/deal-cancellation/check-details';
+import dealsPage from '../../../pages/dealsPage';
+import facilitiesPage from '../../../pages/facilitiesPage';
+import { caseSubNavigation, successBanner } from '../../../partials';
+import activitiesPage from '../../../pages/activities/activitiesPage';
+import { MOCK_APPLICATION_AIN } from '../../../../fixtures/mock-gef-deals';
+import { anIssuedCashFacility } from '../../../../../../e2e-fixtures/mock-gef-facilities';
+import { DEAL_TYPE } from '../../../../../../gef/cypress/fixtures/constants';
+
+context('Deal cancellation - submit cancellation with "effectiveFrom" in past', () => {
+  let dealId;
+  let facility;
+  const ukefDealId = 10000001;
+
+  before(() => {
+    cy.insertOneGefDeal(MOCK_APPLICATION_AIN, BANK1_MAKER1).then((insertedDeal) => {
```

Comment:
These are the same tests as existed originally, but now testing GEF deals instead